pdc/settings_local.py.dist

```diff
@@ -24,6 +24,10 @@
 # checks, uncomment this line, set DISABLE_RESOURCE_PERMISSION_CHECK with True.
 #     DISABLE_RESOURCE_PERMISSION_CHECK = True
 #
+# Example 3: set the sigkey's key_id fd431d51 as default, this default key_id is
+# used when insert a release into database but without a sigkey, it will be set
+# into the release. This default sigkey's name is "redhatrelease2".
+#     RELEASE_DEFAULT_SIGKEY = "fd431d51"
```

Comment:
Remove the key IDs.

I don't like the examples here are used to describe options, it's bit difficult to read. Can you move out and use similar format as for options below (`DEBUG_TOOLBAR`, `BIG_CHANGE_AUTHORS_WHITELIST`)?

It doesn't describe what happens if the key doesn't exist.

Reply:
I am not sure if I can get your point here, I have committed patch for the ansible codes, to set the RELEASE_DEFAULT_SIGKEY there.
And this comment is for other people like fedora guys who want to build a pdc server, they can choose if to set this RELEASE_DEFAULT_SIGKEY.

Reply:
I understand, but why mention these particular key IDs?

Reply:
Aha, this is et's default key, as I have set it in ansible codes, not sure if we can remove this to set a fake one, so keep it here for the review :)

Reply:
I think it's OK to use the same value as for tests (provided the option is commented-out).

Reply:
Have changed the name to a non-existent one "Sigkey's key_id"